### DISCUSSION TITLE
tpot running - still some issues with live attac map

### DISCUSSION THREAD

**Discussion by DO2SN** [telekom-security/tpotce#1584] [General]
With help of tpot-assistant which is a great tool for beginners, I could get up my Tpod "hive" installation running. 

Thanks for this great tool. 

Still have some little issues with Data in live attack map. It shows no Data but "online". 

Live Attack Map
![grafik](https://github.com/telekom-security/tpotce/assets/171075854/6628b404-f456-4092-8fe8-4e751571314e)

Elastic: 

![grafik](https://github.com/telekom-security/tpotce/assets/171075854/c2b75ea8-a510-4bb4-a822-0183ad764d31)

Suricata: 

![grafik](https://github.com/telekom-security/tpotce/assets/171075854/b0dbaadf-969f-4838-9ce8-7730783a21a2)

Status  Container: 

![grafik](https://github.com/telekom-security/tpotce/assets/171075854/a7624d42-4bd9-4d7f-bd15-1dfd9e2aed8b)

Portfreigaben: 

![grafik](https://github.com/telekom-security/tpotce/assets/171075854/2e49a9e0-9bb7-47b5-b324-64fd270171a0)

Ressources: 

![grafik](https://github.com/telekom-security/tpotce/assets/171075854/90491a76-a42a-44bb-b177-eb701bdda071)

Logstach und Kibana Logs 

[Kibana_Logs.txt](https://github.com/user-attachments/files/15876251/Kibana_Logs.txt)

Amazing tool!

**Comment by t3chn0m4g3** [comment]
Thank you for the great feedback on T-Pot Assistant!
The Attack Map needs two valid geo points to render the path from a source to a destination geo point. The dashboard screenshots you provided simply show events from a source based geo point. This was discussed in #1569 and should give you more insight.

**Reply by DO2SN** [reply]
Thanks a lot. It is not a tpot issue. 

I have not a static IP adress and I think there are some settings to do with my fritz box. 

I will look for a solution.
